fetchDependencies

```diff
@@ -512,9 +525,10 @@ if [ "$XC_USE_BCKGND" != "" ]; then
 fi
 
 if [ "$BLD_SPECIFIED" != "" ]; then
-	# Build XCFrameworks, update latest symlink, and clean MoltenVK for rebuild
+	# Build XCFrameworks, update latest symlink, remove intermediates, and clean MoltenVK for rebuild
 	PROJECT_DIR="."
 	CONFIGURATION=${XC_CONFIG}
+	export SKIP_PACKAGING=""
```

Comment:
Since you already `export` `SKIP_PACKAGING` above at top level, it shouldn't be necessary to do so here as well.

Reply:
Done. Thanks.